dataland-frontend/src/components/forms/CreateLksgDataset.vue

```diff
@@ -29,33 +29,35 @@
                 :name="category.name"
               >
                 <div class="uploadFormSection grid" v-for="subcategory in category.subcategories" :key="subcategory">
-                  <div class="col-3 p-3 topicLabel">
-                    <h4 :id="subcategory.name" class="anchor title">{{ subcategory.label }}</h4>
-                    <div :class="`p-badge badge-${category.color}`">
-                      <span>{{ category.label.toUpperCase() }}</span>
+                  <template v-if="subcategoryVisibility.get(subcategory) ?? true">
```

Comment:
Usgin a `<template>` tag here is unnecessary.
The <template> tag in the DOM is typically used in Vue.js for components, directives, or rendering templates, where its contents are dynamically replaced or manipulated by Vue. If you are not using the <template> tag for such purposes and it's not providing any specific benefit, it's better to use an appropriate HTML element like <div>.

Reply:
Ok I found a lot of different opinions in the internet on this. So I think it does not matter that much if you use template or div.
Therefore I accept and close this now.

👍